src/components/tile.js

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
 
-const Tile = ({ enabled, onClick, tileValue }) => {
+const Tile = ({ enabled, onClick, tileImage, tileImageCoords, tileValue }) => {
     const handleClick = () => {
```

Comment:
You should not re-declare function at each render.  Create a "class" or use "withProps" in the following way instead.

```
return compose(
   withProps(({ enabled, onClick, tileValue }) => ({
      handleClick: () => enabled && onClick(tileValue),
   }))
)(Tile);
```